gpii/node_modules/flowManager/src/DefaultSettingsLoader.js

```diff
@@ -44,7 +44,12 @@ fluid.defaults("gpii.defaultSettingsLoader", {
         remoteDefaultSettingsDataSource: {
             type: "kettle.dataSource.URL",
             options: {
-                url: "{defaultSettingsLoader}.options.defaultSettingsUrl"
+                url: "{defaultSettingsLoader}.options.defaultSettingsUrl",
+                components: {
+                    enconding: {
```

Comment:
Misspelling - "encoding"

Reply:
indeed, I even managed myself to duplicate this typo in the commit message :facepalm: